network-api/networkapi/buyersguide/migrations/0008_auto_20210122_0006.py

```diff
@@ -0,0 +1,45 @@
+# Generated by Django 2.2.17 on 2021-01-22 00:06
```

Comment:
this is from a different PR and will disappear on the next wagtail-localize master resync